tests/test_atmos.py

```diff
@@ -624,3 +625,37 @@ def test_late_frost_days(self, atmosds):
         lfd = atmos.late_frost_days(tasmin, date_bounds=("04-01", "06-30"))
 
         np.testing.assert_allclose(lfd.isel(time=0), exp, rtol=1e-03)
+
+
+def test_chill_units(atmosds):
+    tasmax = atmosds.tasmax
+    tasmin = atmosds.tasmin
+    tas = make_hourly_temperature(tasmin, tasmax)
+    cu = atmos.chill_units(tas, date_bounds=("04-01", "06-30"))
+    assert cu.attrs["units"] == "1"
+    assert cu.name == "cu"
+    assert cu.time.size == 4
+
+    exp = [-5029.5, -6634.5, -5993.0, -6596.0, -5654.0]
+    np.testing.assert_allclose(cu.isel(time=0), exp, rtol=1e-03)
+
+
+def test_chill_portions(atmosds):
+    tasmax = atmosds.tasmax
+    tasmin = atmosds.tasmin
+    tas = make_hourly_temperature(tasmin, tasmax)
+    cp = atmos.chill_portions(tas, date_bounds=("09-01", "03-30"), freq="YS-JUL")
+    assert cp.attrs["units"] == "1"
+    assert cp.name == "cp"
+    # Although its 4 years of data its 5 seasons starting in July
+    assert cp.time.size == 5
+
+    # First timestamp is all nan so we test on second
+    exp = [
+        99.91534493319304,
+        96.84276565115084,
+        16.720155001987106,
+        77.86606372811245,
+        140.85516682327875,
+    ]
```

Comment:
I checked the values now and I think there are in the right ball park. Is it expected that for this date_bounds the first and last value are nans @aulemahal ?

Reply:
I checked the code a little I think this is expected for periods that are not complete?

Reply:
Yes it is expected. The data starts on jan 1st and ends on dec 31st. So, with YS-JUL, xarray adds a year before and after that have only 6 months of data. And yout `date_bounds` do not cover this 6 months exactly, so the missing algorithm still sees missing data (sept to dec in the first period, jan to march in the last).

If you have any other tool with which you could compute this indicator and use that for expected values, that would be the best (and mention that source in the test comments).  But otherwise, at least this test will detect future changes in the function or stack, that's better than nothing.

Reply:
I used the chillR package to confirm the values (I now mention it in a comment) the values arent an exact match but in my opinion close enough. The difference could for example be explained by them using `273` to convert to Kelvin.